src/main/java/teammates/ui/webapi/action/GetSessionResultsAction.java

```diff
@@ -25,6 +25,10 @@ protected AuthType getMinAuthLevel() {
 
     @Override
     public void checkSpecificAccessControl() {
+        if (userInfo.isAdmin) {
```

Comment:
is this a related change?

Reply:
This is because I was writing code `GetSessionResultsActionTest` which is currently missing. Although I agree that this is not exactly related to CSV downloads, but still I think the test is nevertheless necessary. And since I'm writing `GetSessionResultsAsCsvActionTest`, I thought it would be ok just to add `GetSessionResultsActionTest` at the same time.